app/components/record_details/view.rb

```diff
@@ -40,7 +42,7 @@ def record_detail_rows
       rows = [provider_row]
       if trainee.requires_lead_partner?
```

Comment:
Can `requires_lead_partner?` be renamed (and the constant it references `LEAD_PARTNER_ROUTES`)?